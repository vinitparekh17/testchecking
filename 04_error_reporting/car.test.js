import Car from './car';

let car;

describe('car', () => {

  beforeEach(() => {
    car = new Car(88);
  });

  it('has correct engine', () => {
    expect(car.engine).toBe('2.94 L');
  });

  it('accelerates', () => {
    car.accelerate();
    expect(car).toMatchSnapshot();
  });

  it('brakes', () => {
    car.brake();
    expect(car.speed).toBe(87);
  });
});
