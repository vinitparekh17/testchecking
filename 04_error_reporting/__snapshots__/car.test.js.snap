// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`car accelerates 1`] = `
Car {
  "engine": "2.85 L",
  "power": "97 kW",
  "speed": 88,
}
`;
